csharp/extractor/Semmle.Extraction/LocationExtensions.cs

```diff
@@ -76,7 +76,7 @@ private static bool BetterThan(Location l1, Location l2)
         /// lexicographically smaller filepath and span is considered better.
         /// </summary>
         public static Location? BestOrDefault(this IEnumerable<Location> locations) =>
-            locations.Any() ? locations.Aggregate((best, loc) => BetterThan(best, loc) ? best : loc) : null;
+            locations.Any() ? locations.Aggregate((best, loc) => BetterThan(loc, best) ? loc : best) : null;
```

Comment:
Would it be difficult to construct a test case where this change makes a difference?

Reply:
Yes, I think so. This last change is just to ensure that we pick the "first" location instead of the last unless there is a better location (and it would have to be non-source locations). Not even sure, how we can achieve that.